test/Performance/PerformanceTestBase.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Microsoft.DotNet.TestFramework;
+
+namespace Microsoft.DotNet.Tools.Builder.Tests
+{
+    public class PerformanceTestBase : TestBase
+    {
+        protected void Build(string project)
```

Comment:
Agreed, I'll inline command executions, just wanted to type less arguments and drop base class too.